src/components/POLine/POLineDetails/POLineDetailsForm.js

```diff
@@ -1,72 +1,273 @@
+      { value: 'Gift', label: intl.formatMessage({ id: 'ui-orders.acquisition_method.gift' }) },
+      { value: 'Purchase At Vendor System', label: intl.formatMessage({ id: 'ui-orders.acquisition_method.purchaseAtVendorSystem' }) },
+      { value: 'Purchase', label: intl.formatMessage({ id: 'ui-orders.acquisition_method.purchase' }) },
+      { value: 'Technical', label: intl.formatMessage({ id: 'ui-orders.acquisition_method.technical' }) }
+    ];
+    const orderFormatDD = [
+      selectPlaceholder,
+      { value: 'Container', label: intl.formatMessage({ id: 'ui-orders.order_format.container' }) },
+      { value: 'Electronic Resource', label: intl.formatMessage({ id: 'ui-orders.order_format.electronicResource' }) },
+      { value: 'P/E Mix', label: intl.formatMessage({ id: 'ui-orders.order_format.PEMix' }) },
+      { value: 'Physical Resource', label: intl.formatMessage({ id: 'ui-orders.order_format.physicalResource' }) },
+      { value: 'Service', label: intl.formatMessage({ id: 'ui-orders.order_format.service' }) },
+    ];
+    const receiptStatusDD = [
+      selectPlaceholder,
+      { value: 'Awaiting Receipt', label: intl.formatMessage({ id: 'ui-orders.receipt_status.awaitingReceipt' }) },
+      { value: 'Cancelled', label: intl.formatMessage({ id: 'ui-orders.receipt_status.cancelled' }) },
+      { value: 'Fully Received', label: intl.formatMessage({ id: 'ui-orders.receipt_status.fullyReceived' }) },
+      { value: 'Partially Received', label: intl.formatMessage({ id: 'ui-orders.receipt_status.partiallyReceived' }) },
+      { value: 'Pending', label: intl.formatMessage({ id: 'ui-orders.receipt_status.pending' }) },
+      { value: 'Receipt Not Required', label: intl.formatMessage({ id: 'ui-orders.receipt_status.receiptNotRequired' }) },
+    ];
+    const workflowStatuses = [
+      selectPlaceholder,
+      { value: 'Closed', label: intl.formatMessage({ id: 'ui-orders.po_line_workflow_status.closed' }) },
+      { value: 'Open', label: intl.formatMessage({ id: 'ui-orders.po_line_workflow_status.open' }) },
+      { value: 'Pending', label: intl.formatMessage({ id: 'ui-orders.po_line_workflow_status.pending' }) },
+    ];
+    const paymentStatuses = [
+      selectPlaceholder,
+      { value: 'Awaiting Payment', label: intl.formatMessage({ id: 'ui-orders.payment_status.awaitingPayment' }) },
+      { value: 'Cancelled', label: intl.formatMessage({ id: 'ui-orders.payment_status.cancelled' }) },
+      { value: 'Fully Paid', label: intl.formatMessage({ id: 'ui-orders.payment_status.fullyPaid' }) },
+      { value: 'Partially Paid', label: intl.formatMessage({ id: 'ui-orders.payment_status.partiallyPaid' }) },
+      { value: 'Payment Not Required', label: intl.formatMessage({ id: 'ui-orders.payment_status.paymentNotRequired' }) },
+      { value: 'Pending', label: intl.formatMessage({ id: 'ui-orders.payment_status.pending' }) },
+    ];
+
```

Comment:
I guess, it would be better to place these constants out into the separate file.
Instead of label with formatMessage() can be used only label-id. The file with constants should be imported into current file. `<Field>` Components should be refactored into new syntax with using `<FormattedMessage>`, so dataOptions will be removed: 
https://github.com/folio-org/ui-eholdings/blob/48bf682afc18f43009c97f43b076f49ebf715700/src/components/package/_fields/content-type/package-content-type-field.js

Reply:
@anastasiaHahanova done, please review again

Reply:
That's great, that you removed all formatMessage(). Thanks!
However, it would be better to not repeat `<FormattedMessage>` in `<Field>`. Better use `.map()`.

Reply:
@anastasiaHahanova done, please check